packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -197,9 +205,9 @@ export interface IContextualMenuItem {
   href?: string;
 
   /**
-   * A collection of submenu items
+   * A collection of submenu items, or an IContextualMenuProps object to apply for the sub menu
    */
-  items?: IContextualMenuItem[];
+  items?: IContextualMenuItem[] | IContextualMenuProps;
 
```

Comment:
It's very easy for us, as developers of this framework, to say that specifying both `items` and `subMenuProps` would be obviously ambiguous. But it may not be as obvious to those less well-versed in the framework. It may even become less obvious to us if we add more of such 'obvious' cases, as I've learned from experience.